### PR TITLE
Fix distribution tests after reachability changes

### PR DESCRIPTION
**Pull request by MatkovIvan**
* [ ] Has `CHANGELOG.md` been updated?
* [x] Are tests passing locally?
* [x] Are the files formatted correctly?
* [x] Did you add unit tests?
* [ ] Did you check UI tests on the sample app? They are not executed on CI.
* [ ] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

Fix distribution tests after reachability changes

### CODE REVIEW

File: AppCenterDistribute/AppCenterDistributeTests/MSDistributeTests.m [hunk 1]
```diff
@@ -1461,12 +1461,15 @@ - (void)testDeleteUpdateTokenAfterReinstall {
...
   // If
-  id reachabilityMock = OCMClassMock([MS_Reachability class]);
-  OCMStub([reachabilityMock reachabilityForInternetConnection]).andReturn(reachabilityMock);
-  OCMStub([reachabilityMock currentReachabilityStatus]).andReturn(NotReachable);
+  self.currentNetworkStatus = NotReachable;
   id distributeMock = OCMPartialMock(self.sut);
   OCMReject([distributeMock buildTokenRequestURLWithAppSecret:OCMOCK_ANY releaseHash:kMSTestReleaseHash isTesterApp:false]);
 
+  // Reject submitting calls.
+  id ingestionCallMock = OCMPartialMock([MSIngestionCall alloc]);
```
Comment (annakocheshkova) [inline AppCenterDistribute/AppCenterDistributeTests/MSDistributeTests.m]: That's the third time I see this block. Maybe a small helper method?

Comment (MatkovIvan) [reply AppCenterDistribute/AppCenterDistributeTests/MSDistributeTests.m]: I guess this is OK for mocking code in tests. We need to be consistent at least.

File: AppCenter/AppCenterTests/MSAppCenterIngestionTests.m [hunk 2]
```diff
@@ -56,7 +56,7 @@ - (void)setUp {
 
 - (void)tearDown {
   [super tearDown];
-
+  [self.reachabilityMock stopMocking];
```
Comment (clpolet) [inline AppCenter/AppCenterTests/MSAppCenterIngestionTests.m]: Good catch :)

File: AppCenterDistribute/AppCenterDistributeTests/MSDistributeTests.m [hunk 3]
```diff
@@ -117,6 +120,19 @@ - (void)setUp {
   self.distributeInfoTrackerMock = OCMClassMock([MSDistributeInfoTracker class]);
   self.sut.distributeInfoTracker = self.distributeInfoTrackerMock;
 
+  // Mock reachability.
+  self.reachabilityMock = OCMClassMock([MS_Reachability class]);
```
Comment (clpolet) [inline AppCenterDistribute/AppCenterDistributeTests/MSDistributeTests.m]: I have the feeling that we should create a test utility class dedicated to reachability. The current network status would be a property of that class.

Comment (clpolet) [reply AppCenterDistribute/AppCenterDistributeTests/MSDistributeTests.m]: But it can be a separate work item.

Comment (ElektrojungeMS) [reply AppCenterDistribute/AppCenterDistributeTests/MSDistributeTests.m]: I agree that having a separate work item makes sense.

**Comment by ElektrojungeMS** [comment]
I've created a separate work item for moving reachability related code to a util class for the tests. Let's merge this!

**Pull request merged**